environment/create_google_config.py

```diff
@@ -0,0 +1,308 @@
+def find_billing_accounts():
+	# from https://github.com/lukwam/gcp-tools/blob/master/lib/google.py#L216
+	# create a request to list billingAccounts
+    billing_accounts = billing.billingAccounts()
+    request = billing_accounts.list()
+
+    # create a list to hold all the projects
+    billing_accounts_list = []
+
+    # page through the responses
+    while request is not None:
+
+        # execute the request
+        response = request.execute()
+
+        # add projects to the projects list
+        if 'billingAccounts' in response:
+            billing_accounts_list.extend(response['billingAccounts'])
+
+        request = billing_accounts.list_next(request, response)
+
+    if len(billing_accounts_list) == 0:
+    	# User does not have access to any billing accounts
+    	print "You do not have a Google billing account set up. In order to run\nWDLs in the Google cloud you need an account to bill to. See the README\nfor more details.\nTo learn about creating a billing account, see here: \nhttps://cloud.google.com/billing/docs/how-to/manage-billing-account#create_a_new_billing_account"
+        sys.exit("Exiting.")
+
+    else:
+    	print "\nYou have access to the following Google billing accounts: "
+    	
+    	# Setup table
+    	headers = "Billing Account ID\tBilling Account Name"
+    	print headers
+    	print '-' * len(headers.expandtabs()) 
+
+    	# Iterate and print every billing account 
+    	for billing_acct in billing_accounts_list:
+    		print "%s\t%s" % (billing_acct["name"].replace("billingAccounts/",""), billing_acct["displayName"])
+
+    	print "\nEnter the \"Billing Account ID\" of the billing account you want to use\nto create a new Google project."
+    	#TODO add note that this Google Project is where the compute will run, the billing account is what will be charged when user runs the sample script, or when they use the config. etc
+    	billing_account_id = raw_input("(IDs are case-sensitive and will look similar to this: 002481-B7351F-CD111E): ")
+    	print "You have selected this Billing Account: %s" % billing_account_id
+
+    	# Project name with datetime stamp (minute and second) and user's email address
+    	# DO NOT PUT AN UNDERSCORE '_' IN THE NAME, it cannot be longer than 30 characters, nor can it have "google"
+    	global project_name
+    	project_name = "cromwell-" + "%s" % (check_output(['gcloud', 'config', 'get-value', 'core/account']).rstrip().partition("@")[0]) + datetime.datetime.now().strftime("-%M-%S")
+    	create_google_project(project_name, billing_account_id)
+
+# Create a google project for the user
+def create_google_project(project_name, billing_account_id):
+	# Create google project
+	print "Creating Google project..."
+	body = {'project_id': '%s' % project_name, 'name': '%s' % project_name, 'labels': None}
+	crm.projects().create(body=body).execute()
+	# Sleep is necessary to give time to create project
+	#TODO decrease time?
+	time.sleep(10) 
```

Comment:
10 seconds often won't be enough time - you should change this to checking on the status of this project in a while loop with a sleep in it